# Request
bool ready
bool all_objs
---
# Response
int32[] ids
